bip-0068.mediawiki

```diff
@@ -0,0 +1,199 @@
+<pre>
+  BIP: 68
+  Title: Consensus-enforced transaction replacement signalled via sequence numbers
+  Author: Mark Friedenbach <mark@friedenbach.org>
+  Status: Draft
+  Type: Standards Track
+  Created: 28-05-2015
+</pre>
+
+==Abstract==
+
+This BIP describes a modification to the consensus-enforced semantics of the sequence number field to enable a signed transaction input to remain invalid for a defined period of time after confirmation of its corresponding output, for the purpose of supporting consensus-enforced transaction replacement features.
+
+==Motivation==
+
+Bitcoin has sequence number fields for each coin a transaction is spending. The original idea appears to have been that the highest sequence number should dominate and miners should prefer it over lower sequence numbers. This was never really implemented, and the half implemented code seemed to be making this assumption that miners would honestly prefer the higher sequence numbers, even if the lower ones were much much more profitable. That turns out to be a dangerous assumption, and so most technical people have assumed that kind of sequence number mediated replacement was useless, because there was no way to enforce "honest" behaviour, as even a few rational (profit maximizing) miners would break that completely. The change described by this BIP provides the missing piece that makes sequence numbers do something significant with respect to enforcing transaction replacement without assuming anything other than profit-maximizing behaviour on the part of miners.
+
+==Specification==
+
+The maximum sequence number can be included in any block, like normal. For transactions with an nVersion of 2 or greater, a sequence number of one less than the maximum can only be included in the next block after the input it is spending, rather than it being possible to be included in the same block. A sequence number one less than that can only be included two blocks later, and so on. Alternatively, a sequence number LOCKTIME_THRESHOLD (500,000,000) less than the maximum (0xffffffff - 500,000,000 = 0xe2329aff can only be included in a block with an nTime timestamp at least one second greater than the median time past of the block of the input it is spending. A sequence number one less than that can only be included two seconds later, and so on. This behaviour is only enforced if the most significant bit of the sequence number field is set.
```

Comment:
> (0xffffffff - 500,000,000 = 0xe2329aff

Unclosed parens

> one second greater than the median time past of the block of the input it is spending.

"median time past" looks like a normal adjective phrase, but it makes no sense in context unless you know that's the name of a Bitcoin Core function.  I suggest either placing it in italics to indicate foreign phrasing, using `GetMedianTimePast()`, or expanding it to describe what `GetMedianTimePast()` does.